.github/workflows/actions_build.yml

```
@@ -190,3 +190,40 @@ jobs:
+  docker-build:
+    if: github.repository == 'line/centraldogma'
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    env:
+      GRADLE_OPTS: -Xmx1280m
+    steps:
+      - uses: actions/checkout@v3
+
+      - id: setup-jdk
+        name: Set up JDK ${{ env.BUILD_JDK_VERSION }}
+        uses: actions/setup-java@v3
```

actionlint (1.7.8)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)